model/delivery/form/NoLtiProviderException.php

```diff
@@ -1,43 +1,25 @@
 <?php
```

Comment:
Pretty weird file conversion... RegisterTaoConsumer is used in manifest. Has it been deleted?

Reply:
This is a new file, the original one has been deleted

Reply:
Ok. It just showed as renamed to NoLtiProviderException in github previously, now it shows up correctly.